### PR TITLE
minor: fix typos and minor grammar issues in README

### PR DESCRIPTION
**Pull request by paullessing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,18 +49,18 @@ and then run Docker Compose as follows: ``docker-compose -f docker-compose.yml -
 If you want to enable document sharing via [Etherpad], configure it and run Docker Compose as
 follows: ``docker-compose -f docker-compose.yml -f etherpad.yml up``
 
-If you want to use jibri too, first configure a host as described in JItsi BRoadcasting Infrastructure configuration section
+If you want to use jibri too, first configure a host as described in Jitsi Broadcasting Infrastructure configuration section
```
Comment (sapkra) [inline README.md:52]: This is intentional I think because it symbolizes Jibri -> JI BR I

Comment (paullessing) [reply README.md]: Thanks, good spot. Undid the change

**Comment by saghul** [comment]
Oops looks like this has some conflicct now, can you please take care of it?

**Comment by paullessing** [comment]
Updated with merge.

**Pull request merged**